cli/commands/login.go

```diff
@@ -168,3 +206,13 @@ func (cmd *Login) Run(scope scope.Scope, c *cli.Context) {
 	}
 	fmt.Printf("Connected to Brooklyn version %s at %s\n", loginVersion.Version, cmd.network.BrooklynUrl)
 }
+
+func (cmd *Login) isBasicAuth() bool {
+	return strings.EqualFold(cmd.network.AuthorizationType,BASIC_AUTH)
```

Comment:
this can just be simply `cmd.network.AuthorizationType == BASIC_AUTH` and doesn't really need to be in a separate function.